.pre-commit-config.yaml

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
I guess it must have been a typo, can you leave this file as it was?

Reply:
Hello @chienandalu can you check this? It gives us a conflict when doing the pull in the gitaggregattor to test all the PRs.

Reply:
We're not having this issue in our gitaggregator setup. Anyway, is a pre-commit thing. When I remove the change, pre-commit recovers it back... Maybe I should remove upstream or simply just merge this (it's been working in prod for months) and rebase the others....

Reply:
It seems that you don't have the repo updated.
![image](https://github.com/user-attachments/assets/64a31859-fd4d-4a00-a42b-7ef6e68c88de)

Reply:
Yeah. That was it :) Thanks!